Add `.css.cjs` as a matching Vanilla Extract file (#1031)
vanilla-extract-css/vanilla-extract: .changeset/big-grapes-juggle.md

---
'@vanilla-extract/integration': patch
---

> Add `.css.cjs` as a matching Vanilla Extract file

Reviewer: ```suggestion
Add `.css.cjs` extension to the Vanilla Extract file filter
```
